samples/excel/85-preview-apis/range-get-range-edge.yaml

```
@@ -0,0 +1,229 @@
+order: 8
+id: excel-get-range-edge
+name: Select used range edge
+description: This sample shows two ways to select the edge of the used range, based on the currently selected range.
```

Suggestion to make this description more evergreen.
```suggestion
description: This sample shows how to select the edges of the used range, based on the currently selected range.
```